AsyncDisplayKit/Details/ASCollectionViewFlowLayoutInspector.m

```diff
@@ -67,11 +63,7 @@ - (void)didChangeCollectionViewDelegate:(id<ASCollectionDelegate>)delegate;
 
 - (void)didChangeCollectionViewDataSource:(id<ASCollectionDataSource>)dataSource
 {
-  if (dataSource == nil) {
-    memset(&_dataSourceFlags, 0, sizeof(_dataSourceFlags));
-  } else {
-    _dataSourceFlags.implementsNumberOfSectionsInCollectionView = [dataSource respondsToSelector:@selector(numberOfSectionsInCollectionView:)];
-  }
+  // nop
```

Comment:
Since this delegate method is optional, maybe we should just remove it here and [here](https://github.com/facebook/AsyncDisplayKit/pull/2906/files#diff-7187b8a17e0b7d3d8ec80608f32ac09eR43)?